lv_gif.c

```diff
@@ -119,7 +120,7 @@ static void next_frame_task_cb(lv_task_t * t)
 
     int has_next = gd_get_frame(ext->gif);
     if(has_next == 0) {
-        lv_event_send(img, LV_EVENT_LEAVE, NULL);
```

Comment:
`lv_event_send(img, LV_EVENT_LEAVE, NULL);` should be called to let the user know the gif is ended.

Please send the signal first (to let the system process the event) and then send the event (to let the user know about the event).

Reply:
@kisvegabor Okay, I didn't realize the usage of LV_EVENT_LEAVE, added LV_EVENT_LEAVE back, please check.